waveform_editor/util.py

```diff
@@ -42,6 +43,13 @@ def times_from_csv(source, from_file_path=True):
     return np.array(time_array)
 
 
+class EquilibriumInput(param.Parameterized):
+    """Parameterized class containing an equilibrium URI and time input."""
+
+    equilibrium = param.String(label="Input 'equilibrium' IDS")
```

Comment:
Let's be clear what we mean here:
```suggestion
    uri = param.String(label="URI of the equilibrium IDS")
```